browser/farbling/brave_navigator_useragent_farbling_browsertest.cc

```diff
@@ -140,17 +111,11 @@ IN_PROC_BROWSER_TEST_F(BraveNavigatorUserAgentFarblingBrowserTest,
   // Farbling level: off
   AllowFingerprinting(domain_b);
   NavigateToURLUntilLoadStop(url_b);
-  // HTTP User-Agent header we just sent in that request should be the same as
-  // the unfarbled user agent
-  EXPECT_EQ(last_requested_http_user_agent(), unfarbled_ua);
```

Comment:
I don't think we should be removing these checks. We still want to verify that the network user agent is not farbled in all cases

Reply:
fixed in https://github.com/brave/brave-core/pull/8562/commits/880e03d5e7093cb95ee0e0d50120e142ae34a246